### PR TITLE
Update buildah-config.1.md to include Port/Protocol

### PR DESCRIPTION
**Pull request by ranjithrajaram**
Protocol can be specified with --port option like --port 514/udp. Updating buildah-config documentation to include the protocol option

/kind documentation

With buildah -1.29.1-2 and higher, udp port does get exposed instead of default TCP

[root@atomic-test tmp]# buildah config --port 53/udp $container
[root@atomic-test tmp]# buildah commit --format docker $container port:latest

[root@atomic-test tmp]# podman create -it -P port:latest
56545611f432ad5557e3ef30204e835403a0f51460b108fc2cf32a27fdb06d28
[root@atomic-test tmp]# podman start 56545611f432ad5557e3ef30204e835403a0f51460b108fc2cf32a27fdb06d28
56545611f432ad5557e3ef30204e835403a0f51460b108fc2cf32a27fdb06d28
[root@atomic-test tmp]# podman port -l
53/udp -> 0.0.0.0:5714

### CODE REVIEW

**Comment by rhatdan** [comment]
Thanks @ranjithrajaram 
Please cleanup dangling whitespace.
/approve
LGTM

File: docs/buildah-config.1.md [hunk 1]
```diff
@@ -182,12 +182,15 @@ This option is typically only meaningful when the image's OS is Windows, and is
...
 
-**--port**, **-p** *port*
+**--port**, **-p** *port/protocol*
 
 Add a *port* to expose when running containers based on any images which
 will be built using the specified container. Can be used multiple times.
-If *port* has a trailing `-`, and is already set, then the *port* is removed from the config.
-If the port is set to "-" then all exposed ports settings are removed from the config.
+To specify whether the port listens on TCP or UDP, use "port/protocol". 
+The default is TCP if the protocol is not specified. To expose on both TCP/UDP,
```
Comment (TomSweeneyRedHat) [inline docs/buildah-config.1.md:190]:
```suggestion
The default is TCP if the protocol is not specified. To expose the port on both TCP and UDP, 
```

File: docs/buildah-config.1.md [hunk 2]
```diff
@@ -182,12 +182,15 @@ This option is typically only meaningful when the image's OS is Windows, and is
...
-**--port**, **-p** *port*
+**--port**, **-p** *port/protocol*
 
 Add a *port* to expose when running containers based on any images which
 will be built using the specified container. Can be used multiple times.
-If *port* has a trailing `-`, and is already set, then the *port* is removed from the config.
-If the port is set to "-" then all exposed ports settings are removed from the config.
+To specify whether the port listens on TCP or UDP, use "port/protocol". 
+The default is TCP if the protocol is not specified. To expose on both TCP/UDP, 
+specify the port option multiple times. If *port* has a trailing `-`, and is already set,
```
Comment (TomSweeneyRedHat) [inline docs/buildah-config.1.md:191]:
```suggestion
specify the port option multiple times. If *port* has a trailing `-` and is already set, 
```

File: docs/buildah-config.1.md [hunk 3]
```diff
@@ -182,12 +182,15 @@ This option is typically only meaningful when the image's OS is Windows, and is
...
+**--port**, **-p** *port/protocol*
 
 Add a *port* to expose when running containers based on any images which
 will be built using the specified container. Can be used multiple times.
-If *port* has a trailing `-`, and is already set, then the *port* is removed from the config.
-If the port is set to "-" then all exposed ports settings are removed from the config.
+To specify whether the port listens on TCP or UDP, use "port/protocol". 
+The default is TCP if the protocol is not specified. To expose on both TCP/UDP, 
+specify the port option multiple times. If *port* has a trailing `-`, and is already set, 
+then the *port* is removed from the config. If the port is set to "-" then all exposed
```
Comment (TomSweeneyRedHat) [inline docs/buildah-config.1.md:192]:
```suggestion
then the *port* is removed from the configuration. If the port is set to "-" then all exposed 
```

File: docs/buildah-config.1.md [hunk 4]
```diff
@@ -182,12 +182,15 @@ This option is typically only meaningful when the image's OS is Windows, and is
...
 
 Add a *port* to expose when running containers based on any images which
 will be built using the specified container. Can be used multiple times.
-If *port* has a trailing `-`, and is already set, then the *port* is removed from the config.
-If the port is set to "-" then all exposed ports settings are removed from the config.
+To specify whether the port listens on TCP or UDP, use "port/protocol". 
+The default is TCP if the protocol is not specified. To expose on both TCP/UDP, 
+specify the port option multiple times. If *port* has a trailing `-`, and is already set, 
+then the *port* is removed from the config. If the port is set to "-" then all exposed 
+ports settings are removed from the config.
```
Comment (TomSweeneyRedHat) [inline docs/buildah-config.1.md:193]:
```suggestion
ports settings are removed from the configuration.
```

**Comment by rhatdan** [comment]
@ranjithrajaram Please squash your commits and sign them

git rebase -i origin 
git commit -a --amend -s
git push --force

**Review by TomSweeneyRedHat — COMMENTED** [review]
@ranjithrajaram, thanks for the touch-ups, especially since most of them were pre-existing.

LGTM

**Comment by ranjithrajaram** [comment]
I will close this request and redo it.

**Pull request closed without merging**